pom.xml

```diff
@@ -46,7 +46,7 @@
 	</modules>
 
     <properties>
-        <openMRSVersion>2.5.6</openMRSVersion>
+        <openMRSVersion>2.5.7</openMRSVersion>
```

Comment:
fwiw just committed this upgrade to PIH Core... for some reason it wasn't picking it up automatically, but everything seemed to run fine when I built locally against 2.5.7